oidc_provider/lib/utils/token.py

```diff
@@ -1,22 +1,30 @@
-from datetime import timedelta
 import time
 import uuid
+from datetime import timedelta
 
 from Cryptodome.PublicKey.RSA import importKey
+from django.contrib.auth.hashers import PBKDF2PasswordHasher
 from django.utils import dateformat, timezone
+from django.utils.crypto import constant_time_compare
 from jwkest.jwk import RSAKey as jwk_RSAKey
 from jwkest.jwk import SYMKey
 from jwkest.jws import JWS
 from jwkest.jwt import JWT
-
-from oidc_provider.lib.utils.common import get_issuer, run_processing_hook
-from oidc_provider.lib.claims import StandardScopeClaims
-from oidc_provider.models import (
-    Code,
-    RSAKey,
-    Token,
-)
 from oidc_provider import settings
+from oidc_provider.lib.claims import StandardScopeClaims
+from oidc_provider.lib.utils.common import get_issuer, run_processing_hook
+from oidc_provider.models import Code, RSAKey, Token
+
+
+class TokenHasher:
+    hasher = PBKDF2PasswordHasher()
```

Comment:
@johnguant you thought this should be SHA256 right? Is that a better fit?

Reply:
Yeah, for a long random token SHA256 is a good fit.